src/PJ_helmert.c

```diff
@@ -46,7 +48,7 @@ Last update: 2016-11-24
 #include <assert.h>
 #include <stddef.h>
 #include <errno.h>
-PROJ_HEAD(helmert, "3- and 7-parameter Helmert shift");
+PROJ_HEAD(helmert, "3-, 7- and 14-parameter Helmert shift");
```

Comment:
For future work, we should also add the 4-parameter 2D Helmert transform (note to self. Will add to issue tracker)